styleguide.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   components: 'src/workflow.tsx',
   propsParser: require('react-docgen-typescript').withDefaultConfig().parse,
+  serverPort: 3000,
```

Comment:
Setting this port doesn't work when running `npm run start:dev`, because it's being used.

Reply:
Do you mean it is clashing with another app running in your docker container? We should change it to a port that is exposed in the container would `8080` work?